examples/set-namespace/helloworld/README.md

```diff
@@ -0,0 +1,76 @@
+Set Namespace
+==================================================
+
+Function `set-namespace` sets namespaces in KRM objects.
+
+## Get Image
+
+If you want to use the cutting-edge version of this function, you need to clone
+the entire repo locally and then run the following to build the image:
+
+<!-- @buildImage @testMaster -->
+```
+TAG=master
+IMAGENAME=gcr.io/kpt-functions-trusted/set-namespace:$TAG
+docker build -t $IMAGENAME ../../../functions/go/set-namespace/
+```
+
+If you want to use the latest released version:
+
+<!-- @selectImage @testStable -->
+```
+TAG=v0.1
+IMAGENAME=gcr.io/kpt-functions-trusted/set-namespace:$TAG
+```
+
+## Run Function
+
+Let's first get the sample package using kpt:
+
+```
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-namespace/helloworld/configs .
+```
+
+Next, let's create a directory for the results:
+
+<!-- @createResults @testMaster @testStable -->
+```
+rm -rf results
```

Comment:
Instead of create a directory in the repo, I suppose using a temporary directory is a common practice.

Reply:
Sure, will do.